c_glib/test/plasma/test-plasma-client.rb

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class TestPlasmaClient < Test::Unit::TestCase
+  def setup
+    omit("Plasma is required") unless defined?(::Plasma)
+  end
+
+  def test_new
+    assert_nothing_raised do
```

Comment:
We can test connecting to store server by running `plasma_store_server`.
We can get `plasma_store_server` path from `plasma.pc`: `pkg-config --variable=executable plasma`

Reply:
I’ve added test cases. https://github.com/apache/arrow/pull/2848/commits/5c7a85024f33b171d52757eb19f9353d6f85523d#diff-8c8187dfbb3bc22f4933d0622e440b7d
Please confirm my understanding.

Reply:
Thanks!